app/main/views/frameworks.py

```diff
@@ -143,6 +144,8 @@ def download_supplier_pack():
 @login_required
 @flask_featureflags.is_active_feature('GCLOUD7_OPEN')
 def framework_updates():
+    current_app.logger.info("g7updates.viewed: user_id:%s supplier_id:%s",
```

Comment:
Should we just log user_email/supplier_id as part of the logged request data in the supplier app? That should cover the G7 updates and help debugging any issues with the non-200 status responses, which might be hard to do with just a supplier app URL